### PR TITLE
structure for container and component and store

### PR DESCRIPTION
**Pull request by mun625**
seperate map-component and map-container clearly.
And create store.js to manage code clearly.

### CODE REVIEW

**Comment by elegant651** [comment]
LGTM

File: shared/components/map.js [hunk 1]
```diff
@@ -24,16 +24,17 @@ const styles = StyleSheet.create({
...
+          style ={styles.map}
+          initialRegion={{
+          latitude: 37.78825,
+          longitude: -122.4324,
+          latitudeDelta: 0.0922,
+          longitudeDelta: 0.0421
+        }}
+        />
+      </View>
+    );
```
Comment (yuda1124) [inline shared/components/map.js]: is it just warning fix?

**Comment by yuda1124** [comment]
LFRM

File: shared/components/map.js (hunk 1 above)
Comment (mun625) [inline shared/components/map.js]: yes man

**Pull request merged**